src/main/java/baseball/service/Compare.java

```diff
@@ -0,0 +1,33 @@
+package baseball.service;
+
+import java.util.List;
+
+public class Compare {
+    private static int countBall(List<Integer> computerNumbers, String playerNumbers) {
+        int ball = 0;
+
+        for(int i = 0; i < playerNumbers.length(); i++) {
+            if(computerNumbers.contains(Character.getNumericValue(playerNumbers.charAt(i)))) {
```

Comment:
이 부분도 countBall 이 형 변환까지 하는 책임을 가질 필요가 있을까? 라는 의구심을 가져보면 좋을 것 같아요❗ 
형 변환하는 메서드를 따로 만들어 집어넣거나, 받는 매개변수를 int 형으로 받으면 더욱 좋을 것 같습니다🙂

+Character 의 getNumericValue 메서드를 활용해서 형변환을 시켜줬군요❗
```suggestion
            if(computerNumbers.contains(playerNumbers.charAt(i)-'0')) {
```
를 활용하시면, 더욱 짧고 간결한 코드가 가능하답니다 😉